[2025-09-01T23:45:02.117+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-01T23:45:02.145+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: market_table_foreign.foreign_HOSE scheduled__2025-09-01T23:44:00+00:00 [queued]>
[2025-09-01T23:45:02.154+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: market_table_foreign.foreign_HOSE scheduled__2025-09-01T23:44:00+00:00 [queued]>
[2025-09-01T23:45:02.154+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-01T23:45:02.169+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): foreign_HOSE> on 2025-09-01 23:44:00+00:00
[2025-09-01T23:45:02.177+0000] {standard_task_runner.py:63} INFO - Started process 2411235 to run task
[2025-09-01T23:45:02.181+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'market_table_foreign', 'foreign_HOSE', 'scheduled__2025-09-01T23:44:00+00:00', '--job-id', '50049', '--raw', '--subdir', 'DAGS_FOLDER/dag_market_live.py', '--cfg-path', '/tmp/tmpw3d9vcf3']
[2025-09-01T23:45:02.183+0000] {standard_task_runner.py:91} INFO - Job 50049: Subtask foreign_HOSE
[2025-09-01T23:45:02.239+0000] {task_command.py:426} INFO - Running <TaskInstance: market_table_foreign.foreign_HOSE scheduled__2025-09-01T23:44:00+00:00 [running]> on host 1877907bc043
[2025-09-01T23:45:02.341+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='market_table_foreign' AIRFLOW_CTX_TASK_ID='foreign_HOSE' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T23:44:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T23:44:00+00:00'
[2025-09-01T23:45:02.342+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-01T23:45:02.358+0000] {foreign_HOSE.py:17} INFO - Kết nối DB
[2025-09-01T23:45:02.680+0000] {foreign_HOSE.py:42} INFO - Đã lưu foreign_HOSE
[2025-09-01T23:45:02.681+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-01T23:45:02.682+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-01T23:45:02.695+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=market_table_foreign, task_id=foreign_HOSE, run_id=scheduled__2025-09-01T23:44:00+00:00, execution_date=20250901T234400, start_date=20250901T234502, end_date=20250901T234502
[2025-09-01T23:45:02.755+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-01T23:45:02.784+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-01T23:45:02.785+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
